### PR TITLE
tracking-issue: Track Pull requests

### PR DESCRIPTION
**Pull request by tsenart**
This changeset adds pull request support to the tracking-issue tool. It
does a lot of refactorings that benefited the cause, and changes the
library we use for GraphQL requests since the other one didn't support
this use case.

In addition to printing pull requests in the top level of each
assignee's workload, it also nests linked pull requests under issues as
a sub-list.

This was requested by the @sourcegraph/distribution team, specifically @slimsag.

### CODE REVIEW

**Comment by slimsag** [comment]
For context, my use case:

> I primarily will use this when I send PRs that do not have issues created, e.g. for work that was not planned or filed as an issue, but did happen (because it was important, was a refactor, etc).

**Comment by slimsag** [comment]
Very nice! It works very well -- I love that it can nest the PRs under the issues too.

I think it could be useful to prefix the title of PRs with `PR: <title>` in the issue (at least when they aren't nested) -- thoughts?

**Comment by tsenart** [comment]
> I think it could be useful to prefix the title of PRs

I added a :shipit: emoji so that they are visually distinguishable.

File: internal/cmd/tracking-issue/main.go [hunk 1]
```diff
@@ -7,16 +7,16 @@ import (
 	"errors"
```
Comment (unknwon) [inline internal/cmd/tracking-issue/main.go:7]: I kinda remember @tsenart was the code owner of this file... but couldn't track that anymore with current CODEOWNERS file.

File: internal/cmd/tracking-issue/main.go [hunk 2]
```diff
@@ -346,99 +556,209 @@ func listIssues(ctx context.Context, cli *githubv4.Client, org, milestone string
...
 			break
 		}
 	}
 
-	return issues, nil
+	return issues, prs, nil
+}
+
+func listIssuesGraphQLQuery(alias string) string {
+	const searchQuery = `%s: search(first: $%sCount, type: ISSUE, after: $%sCursor query: $%sQuery) {
```
Comment (unknwon) [inline internal/cmd/tracking-issue/main.go:718]: Can be simplified as shown in https://play.golang.org/p/QZ07bKBP4V0

```diff
-	const searchQuery = `%s: search(first: $%sCount, type: ISSUE, after: $%sCursor query: $%sQuery) {
+	const searchQuery = `%[1]s: search(first: $%[1]sCount, type: ISSUE, after: $%[1]sCursor query: $%[1]sQuery) {
```

Comment (tsenart) [reply internal/cmd/tracking-issue/main.go:718]: Applied in master.

**Pull request merged**